config/settings/test.yml

```diff
@@ -96,6 +96,14 @@ bgs:
   client_station_id: 281
   client_username: VAgovAPI
   url: https://internal-dsva-vagov-dev-fwdproxy-1893365470.us-gov-west-1.elb.amazonaws.com:4447
+  # To create a new VCR cassette:
```

Comment:
We sure this is all necessary?
I'm definitely not a VCR pro, but I wanna say that if you call ` VCR.use_cassette('bgs/some_totally/random_file') do` (without pre-creating an empty cassette), that it will generate the cassette for you and populate it.